Makefile

```diff
@@ -87,9 +87,9 @@ coverage-filecheck-tests:
 	lit -v tests/filecheck/ -DCOVERAGE
 
 # generate html coverage report
-coverage-report-html:
+coverage-report-html: coverage
```

Comment:
Can't you let it depend on the output file of the coverage command to circumvent your problem?

You can add a `--data-file=DATAFILE` to the `coverage` recipe to explicitely control the output

Reply:
What is the syntax to depend on a file? I have honestly not a lot of clue about make files.
The file is guaranteed to be `.coverage`, AFAIK.

Reply:
I guess I could do this using some shell cmds, but I would prefer to fix it in a makefile-way.